### PR TITLE
Tweak: Add alignment options for the Tab widget close #11997

### LINKED ISSUE #11997: Alignment options for the "Tab" widget bar
feature request
Be able to align the Tab widget bar to the center or right side

### PR DESCRIPTION
**Pull request by arielk**
When the position of tabs is horizontal we added alignment options: start (default) / center / end / justified 

Center: 
image

Justified:
image

End:
image

### CODE REVIEW

**Comment by rodolphebertozzo** [comment]
Good job !

File: includes/widgets/tabs.php [hunk 1]
```diff
@@ -161,6 +161,36 @@ protected function _register_controls() {
 			]
 		);
 
+		$this->add_control(
+			'tabs_align',
+			[
+				'label' => __( 'Alignment', 'elementor-pro' ),
```
Comment (rodolphebertozzo) [inline includes/widgets/tabs.php:167]: Why 'elementor-pro' ?

Comment (arielk) [reply includes/widgets/tabs.php]: thanks, fixed

File: assets/dev/scss/frontend/widgets/tabs.scss [hunk 2]
```diff
@@ -64,6 +64,35 @@
 		}
 	}
 
+	&.elementor-tabs-alignment- {
+
+		&center {
+
+			.elementor-tabs-wrapper {
+				display: flex;
```
Comment (kobizz) [inline assets/dev/scss/frontend/widgets/tabs.scss:72]: `.elementor-tabs-wrapper` can always be in `display: flex`, then you will only need to write it once

File: includes/widgets/tabs.php [hunk 3]
```diff
@@ -285,6 +315,34 @@ protected function _register_controls() {
...
+						'title' => __( 'Center', 'elementor' ),
+						'icon' => 'eicon-text-align-center',
+					],
+					'right' => [
+						'title' => __( 'Right', 'elementor' ),
+						'icon' => 'eicon-text-align-right',
+					],
+				],
+				'selectors' => [
+					'{{WRAPPER}} .elementor-tab-title' => 'text-align: {{VALUE}};',
```
Comment (kobizz) [inline includes/widgets/tabs.php:338]: It might be better to use a CSS variable as our new approach

**Review by kobizz — CHANGES_REQUESTED** [review]

File: assets/dev/scss/frontend/widgets/tabs.scss (hunk 2 above)
Comment (arielk) [reply assets/dev/scss/frontend/widgets/tabs.scss:74]: Fix: https://github.com/elementor/elementor/pull/12002/commits/b641bb3a5885ed4da6dd63ff79138ef28aa10d9f

File: includes/widgets/tabs.php (hunk 3 above)
Comment (arielk) [reply includes/widgets/tabs.php:338]: This widget still does not work in the new method, I prefer to not do it only in one control.

**Review by kobizz — APPROVED** [review]

**Pull request merged**